docs/source/development/index.rst

```diff
@@ -1,7 +1,7 @@
 Development
 ===========
 
-This page describes StackStorm development process and contains general
+This page describes |st2| development process and contains general
```

Comment:
We DON"T opensource BWC thus here in 'contribution/development' we better say "StackStorm".